### PR TITLE
Pin to a default alpine version (3.11) for _most_ images

### PR DESCRIPTION
**Pull request by tobybellwood**
Initial draft of a way to pin a "default" Alpine version - currently 3.11 - for those images that currently utilise :latest.

It also pins specific older versions of alpine for specific images (php, node and python).

Sumbitting a draft to put it through testing - images build ok locally, but not tested (why use my CPU cycles!)

# Checklist
- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated.
- [ ] Changelog entry has been written

# Changelog Entry

# Closing issues
Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Comment by tobybellwood** [comment]
List of issues for follow up
* mariadb (alpine3.8) - 3.10 seems incompatible with api_db [DEBUG]
* varnish (alpine3.7) - 3.10 has varnish 6.2 - needs libvmod plugin updates (dynamic from same repo, bodyaccess now via varnish-modules) [UPDATE]
* mongodb (alpine3.9) - no longer available in 3.10 [WORKAROUND?]

some upstream images don't permit alpine version pinning in the dockerfile - these have been semver pinned to minor releases
* nginx,postgres,rabbitmq - version documented in Dockerfile
* solr - alpine no longer supported as official image (stretch/slim only)

File: images/rabbitmq/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/commons as commons
-FROM rabbitmq:3-management
+# alpine 3.10 as per https://github.com/docker-library/rabbitmq/blob/master/3.8/alpine/Dockerfile
```
Comment (tobybellwood) [inline images/rabbitmq/Dockerfile]: Now 3.11 (Jan 2019) - https://github.com/docker-library/rabbitmq/pull/384
```suggestion
# alpine 3.11 as per https://github.com/docker-library/rabbitmq/blob/master/3.8/alpine/Dockerfile
```

File: images/nginx/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/commons as commons
-FROM openresty/openresty:alpine
+# alpine 3.9 per https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile
```
Comment (tobybellwood) [inline images/nginx/Dockerfile]: As per https://github.com/openresty/docker-openresty/issues/135
```suggestion
# alpine 3.11 per https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile
```

File: images/mongo/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/commons as commons
-FROM alpine:3.8
+FROM alpine:3.9
```
Comment (tobybellwood) [inline images/mongo/Dockerfile]:
```suggestion
FROM alpine:3.8
```
Reverting back to 3.8 to avoid upgrades to images as part of this process.

**Review by Schnitzel — APPROVED** [review]
wow very nice :)

**Pull request merged**